apps/web/src/design-system/button/Button.tsx

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
-import { Button as MantineButton, SharedButtonProps } from '@mantine/core';
+import { Button as MantineButton } from '@mantine/core';
 
 import useStyles from './Button.styles';
-
 import { SpacingProps } from '../shared/spacing.props';
 
 export type Size = 'md' | 'lg' | undefined;
 
 interface IButtonProps extends JSX.ElementChildrenAttribute, SpacingProps {
   loading?: boolean;
   size?: Size;
-  variant?: 'outline' | 'filled';
+  variant?: 'outline' | 'filled' | 'gradient';
```

Comment:
So maybe the filled can be removed? Up until now, the filled varient was our gradient one